### PR TITLE
Certified connector/telnyx sms conn

### PR DESCRIPTION
**Pull request by fabianp5060**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by v-EgorKozhadei** [comment]
Hello Partners, 

Thank you very much and congratulations on your new submission!

Your PR contains changes for one of connector's files, other connectors should not be affected when submitting a new PR. Please remove these changes.
After that our Certification Engineers will review it within 1-2 weeks. 

Please let us know if you have any questions. 

Thank you very much again for working with us.

**Comment by v-EgorKozhadei** [comment]
Hello Partners, 

Thank you very much and congratulations on your new submission!

Your PR contains changes for one of connector's files, other connectors should not be affected when submitting a new PR. Please remove these changes.
After that our Certification Engineers will review it within 1-2 weeks. 

Please let us know if you have any questions. 

Thank you very much again for working with us.

File: certified-connectors/Telnyx SMS/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "publisher": "Telnyx"
```
Comment (v-EgorKozhadei) [inline certified-connectors/Telnyx SMS/apiProperties.json:20]: @fabianp5060 the stackOwner property is missing, please refer to the [apiProperties.json template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json). The current iconBrandColor is [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).

File: certified-connectors/Telnyx SMS/readme.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Telnyx
+Telnyx Suite is a comprehensive cloud communication platform that provides communication capabilities via voice, SMS, fax, and IP services. Telnyx provides global high-speed connections for low latency, high uptime service. The Telnyx Suite includes SIP trunking, messaging API, voice API, lookup API, programmable fax API, and wireless services.
+
+## Publisher: ********
```
Comment (v-EgorKozhadei) [inline certified-connectors/Telnyx SMS/readme.md:4]: @fabianp5060 the publisher's name should follow the `Publisher: Publisher's Name` pattern.

File: certified-connectors/Telnyx SMS/readme.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+## Supported Operations
+### Send a message
+Send a message with a Phone Number, Alphanumeric Sender ID, Short Code or Number Pool.
+
+
+## API Documentation
+https://developers.telnyx.com/
+
+## Known Issues and Limitations
+None.
```
Comment (v-EgorKozhadei) [inline certified-connectors/Telnyx SMS/readme.md:18]: @fabianp5060 please review the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1), all of the required headings should be present.

File: certified-connectors/Telnyx SMS/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v2/messages": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Telnyx SMS/apiDefinition.swagger.json:39]: @fabianp5060 can you please provide a more detailed description for 2xx response codes? You can refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

File: certified-connectors/Telnyx SMS/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "object",
+                  "properties": {
+                    "completed_at": {
+                      "type": "string",
+                      "description": "completed_at"
```
Comment (v-EgorKozhadei) [inline certified-connectors/Telnyx SMS/apiDefinition.swagger.json:48]: @fabianp5060 please provide a bit more detailed descriptions. You can refer to [parameter summaries and descriptions guideline](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions) and to [English strings guideline](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @fabianp5060! Please check comments after the review, thank you!

**Comment by v-EgorKozhadei** [comment]
Hello @fabianp5060, 

I hope you are doing well. 

I am following up with you to see how things are going with preparing your connector. 

Please let us know if you have any questions. 

Thank you.

File: certified-connectors/Telnyx SMS/apiProperties.json (hunk 1 above)
Comment (fabianp5060) [reply certified-connectors/Telnyx SMS/apiProperties.json]: updated in commit [a415100](https://github.com/microsoft/PowerPlatformConnectors/pull/1711/commits/a415100a41164af6aff5ec47643e12a8f844a2b1)

File: certified-connectors/Telnyx SMS/readme.md (hunk 3 above)
Comment (fabianp5060) [reply certified-connectors/Telnyx SMS/readme.md]: resolved in commit 94d2e98

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @fabianp5060! Thank you very much for all the updates! There's one last comment and we're good to go.

**Review by v-EgorKozhadei — APPROVED** [review]

File: certified-connectors/Telnyx SMS/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,260 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Telnyx",
+    "description": "Deliver SMS messages over the Telnyx network to ensure your notifications reach the intended recipient in a fast, reliable, and cost effective manner.",
+    "version": "1.0",
+    "contact":{
+	    "name": "Telnyx Support",
```
[the remark this answers is not in the document]
Comment (fabianp5060) [reply certified-connectors/Telnyx SMS/apiDefinition.swagger.json]: @v-EgorKozhadei I don't see the spacing issue in my source file and ide, but I did see some spacing inconsistencies with the "x-ms-connector-metadata" object.  I have corrected those (and a minor spacing in contact) and have pushed those changes in commit 4f5b131.

Hopefully that resolves the issue

**Comment by v-EgorKozhadei** [comment]
Hello @fabianp5060, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) using your latest commit id, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions. 

Thank you very much for working with us.

**Pull request merged**